app/controllers/organizations/organization_profiles_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Organizations::OrganizationProfilesController < ApplicationController
```

Comment:
To ensure proper organization setup, could we inherit from `Organizations::BaseController`? This will provide us with some useful safeguards related to organization handling 🛡️

Reply:
Very nice catch! :fire: